src/components/Rubric.js

```diff
@@ -50,11 +51,7 @@ const CompetencyButton = withRouter(props => {
               answer={criteria.answer}
               id={criteria.id}
               text={criteria.text} />
-          }).sort((a , b) => {
```

Comment:
removing the sorts here, why? don't you need them anymore? 

in any case, one of the things I forgot to say in the last review was that you could make the sorts be a single fn that you reuse.. but now that you've deleted them, you don't need that anymore?

Reply:
yep. We don't need sort function anymore

Reply:
Sure. I will revert those changes.